api/basic/helper/logger.ts

```diff
@@ -2,18 +2,23 @@ import pino from 'pino'
 import { LambdaContext, LambdaEvent, lambdaRequestTracker, pinoLambdaDestination } from 'pino-lambda'
 
 const destination = pinoLambdaDestination()
-export const logger = pino(destination)
+export const logger = pino({
+  serializers: {
+    err: (e) => `[${e.code || e.constructor.name}] ${e.message}\n${e.stack}`
+  }
+}, destination)
 
-export const withLambdaRequest = lambdaRequestTracker({
-  requestMixin: (event: LambdaEvent, _context?: LambdaContext) => {
+export const setLoggerWithLambdaRequest = lambdaRequestTracker({
+  requestMixin: (event: LambdaEvent, context: LambdaContext) => {
     const cid = event.pathParameters?.cid ?? ''
```

Comment:
instead of picking specific  value we probably should just log `pathParameter` and `queryStringParameters`